src/Polling/Frequent/FrequentPollingActivity.cs

```diff
@@ -0,0 +1,44 @@
+using Microsoft.Extensions.Logging;
+using Temporalio.Activities;
+using Temporalio.Exceptions;
+using TemporalioSamples.Polling.Common;
+
+namespace TemporalioSamples.Polling.Frequent;
+
+public class FrequentPollingActivity : IPollingActivity
+{
+    private static int pollDurationSections = 1;
```

Comment:
Some samples you make the interval a static variable and some you just embed it in the code.